### PR TITLE
Make license header conform to bitExpert defaults

### PR DESCRIPTION
**Pull request by shochdoerfer**

### CODE REVIEW

**Comment by szepeviktor** [comment]
Oh noooooooooooo! :)

```php
/*
 * This file is part of the Composer-PHIVE Bridge package.
 *
 * (c) bitExpert AG
 *
 * For the full copyright and license information, please view the LICENSE
 * file that was distributed with this source code.
 */
```

Good old tags from PEAR come to rescue.

```php
<?php

/**
 * PhiveHandler.php
 *
 * @package bitexpert/composer-phive-bridge
 * @copyright bitExpert AG
 * (or) @author bitExpert AG
 * @license https://github.com/bitExpert/composer-phive-bridge/blob/main/LICENSE.md MIT
 */
 ```

Please see https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/index.html#tag-reference

**Comment by heiglandreas** [comment]
The Doctags are a possible extension (machine-readable) to the human readable License info that has to be part of the file. Sadly otherwise people could object that they don't know what those tags are about and they didn't know that they were meant for them to read and act appropriately. Legal MumboJumbo 🤷‍♂️

File: src/PhiveHandler.php [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-
-/**
- * Copyright bitExpert
+/*
+ * This file is part of the Composer-PHIVE Bridge package.
+ *
+ * (c) bitExpert AG
  *
- * Licenses under the MIT-license. For details see the included file LICENSE.md
+ * For the full copyright and license information, please view the LICENSE
```
Comment (szepeviktor) [inline src/PhiveHandler.php:8]:
```suggestion
 * For the full copyright and license information, please view the LICENSE.md
```

File: src/Plugin.php [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
-
-/**
- * Copyright bitExpert AG
+/*
+ * This file is part of the Composer-PHIVE Bridge package.
+ *
+ * (c) bitExpert AG
  *
- * Licenses under the MIT-license. For details see the included file LICENSE.md
+ * For the full copyright and license information, please view the LICENSE
```
Comment (szepeviktor) [inline src/Plugin.php:8]:
```suggestion
 * For the full copyright and license information, please view the LICENSE.md
```

**Comment by szepeviktor** [comment]
> otherwise people could object that they don't know what those tags are about

Those tags happen to be complete English words.
Anyone understanding _For the full copyright and license information, please view the LICENSE.md_ is able to understand `@license https://github.com/bitExpert/composer-phive-bridge/blob/main/LICENSE.md MIT`
Do you think the `@` sign confuses anyone?

**Comment by szepeviktor** [comment]
> otherwise people could object that they don't know what those tags are about

Following this thinking _(c) bitExpert AG_ may also cause confusion. _Copyright bitExpert AG_ should be written to be 100% clear.

**Comment by szepeviktor** [comment]
I let you operate the way you want.

**Pull request merged**